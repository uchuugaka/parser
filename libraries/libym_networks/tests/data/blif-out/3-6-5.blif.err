"3-6-5.blif" line: 7, column: 15 [ER_MLTDEF01]: x: Defined more than once. Previsous Definition is "3-6-5.blif" line: 6, column: 15.
Error in reading 3-6-5.blif
