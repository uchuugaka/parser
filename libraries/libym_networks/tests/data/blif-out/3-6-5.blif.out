.model test1
	["3-6-5.blif" line: 3, columns:1-6]
	["3-6-5.blif" line: 3, columns:8-12]
  inputs elem: a
	["3-6-5.blif" line: 4, column: 9]
  inputs elem: b
	["3-6-5.blif" line: 4, column: 11]
  outputs elem: x
	["3-6-5.blif" line: 5, column: 10]
.gate inv4x begin
	["3-6-5.blif" line: 6, columns:1-5]
	["3-6-5.blif" line: 6, columns:7-11]
  .gate assign: o = x
	["3-6-5.blif" line: 6, column: 13]
	["3-6-5.blif" line: 6, column: 15]
  .gate assign: a = b
	["3-6-5.blif" line: 6, column: 17]
	["3-6-5.blif" line: 6, column: 19]
.gate end

.gate inv4x begin
	["3-6-5.blif" line: 7, columns:1-5]
	["3-6-5.blif" line: 7, columns:7-11]
error_exit
